Trend Micro/deep-security/ingest/parser.yml

```diff
@@ -75,7 +75,7 @@ stages:
           "25": "Add/Remove Programs"
           "26": "Services"
         mapping:
-          parsed_event.message.cs4: trendmicro.ResourceType
+          parsed_event.message.get("Resource Type"): trendmicro.ResourceType
```

Comment:
I'm not sure this notation will work.
I think the CEF stage should be fixed to replace spaces with an underscore.

Reply:
Hello @squioc, please review my latest commit where I tried to use an intermediate field to bypass this limitation. 

I will try to improve this work once I have more events.